geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/serial/SerialGatewaySenderQueueDUnitTest.java

```diff
@@ -56,6 +58,85 @@
 @Category({WanTest.class})
 public class SerialGatewaySenderQueueDUnitTest extends WANTestBase {
 
+  @Test
+  public void unprocessEventsLinger() throws Exception {
```

Comment:
would we be able to have a better test name?

Reply:
Done!